snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -110,6 +110,68 @@ public void CreateBucket()
         }
 
         [Fact]
+        public void UpdateBucket()
+        {
+            var projectId = _fixture.ProjectId;
+            var setupClient = StorageClient.Create();
+
+            // GCS bucket names must be globally unique
+            var bucketName = Guid.NewGuid().ToString();
+            setupClient.CreateBucket(projectId, bucketName);
+
+            // <Service.Buckets.Update>
+            var client = StorageClient.Create();
+            var bucket = client.Service.Buckets.Get(bucketName).Execute();
```

Comment:
Yes, absolutely. We already have a snippet for that - I just haven't implemented it yet.